README.md

```diff
@@ -61,7 +61,7 @@ For the small price of 8kb gziped. &nbsp;&nbsp; [🏁Get started now](https://re
 
 ## Features
 
-- [x] ![TS](./scripts/typescript.svg?sanitize=true) Strong [Typescript](https://www.typescriptlang.org/) types
+- [x] ![TS](./packages/rest-hooks/typescript.svg?sanitize=true) Strong [Typescript](https://www.typescriptlang.org/) types
```

Comment:
did you remove the scripts folder?

Reply:
just has testSetup now. I had originally put the svg in there to try to make there not be too many top-level files, but now that it's in packages, it's better served there